packages/chrome/src/index.tsx

```diff
@@ -51,7 +59,7 @@ const CopyTheme = ({ theme }) => {
 
 const Spacer = () => <div sx={{ my: 2 }} />
 
-const App = () => {
+const App: FunctionComponent = () => {
```

Comment:
`FunctionComponent` here is a bit redundant.

1. It states that this component takes children. (Try destructuring in props.)
2. We don't use `defaultProps` nor `propTypes` here.

Reply:
Yeah that's fair enough but FunctionComponent is the default's React type for function based components, children (and other props) are optional too.